Developed Android Org App

dlet24 opened the pull request:
Hi @zakame, Here's the PR for the coding test. And I sent an email to your email **zakame@zakame.net** regarding the answers on the question for this test. 

Best regards!

dlet24 committed:
This is the developed app for the coding exam. I made it as simple as possible. It includes a proguard configuration in order to compress the release a little bit more.

dlet24 committed:
Changed the app_release.apk

zakame commented:
The APK file itself should not be in PR :)  Reviewers should be able to build it on our own.

dlet24 committed:
Removed APK file.

dlet24 commented:
Hi @zakame, yes it shouldn't be there. Thank you for reminding me. I'll take note of that. =) 
I updated this PR and removed the APK file.

Have a great day!
